.gitignore

```diff
@@ -1,2 +1,18 @@
-dist
-build
+*.py[cod]
+*$py.class
+*.egg-info
+*.log
+*.pot
+.DS_Store
+.coverage/
+.eggs/
+.idea/
+.project/
+.pydevproject/
+.vscode/
```

Comment:
IMO all those editor-specific things should be in the global gitignore of each developer. Probably a bit late now...